docs/commands/env.md

```diff
@@ -149,10 +149,22 @@ netlify env:set
 
 **Flags**
 
+- `context` (*production | deploy-preview | branch-deploy | dev*) - Specify a deploy context (default: all contexts)
```

Comment:
There is a slight difference -- `all` is no longer in the enum, in order to prevent someone from doing `netlify env:set KEY value --context dev production all` (variadic options are supported, but providing multiple contexts that include `all` doesn't make sense). By saying `all contexts` (general) instead of `"all"` (literal) was my (probably too subtle) way of indicating that.